prov/sm2/src/sm2_fifo.h

```diff
@@ -241,4 +241,30 @@ static inline void sm2_fifo_write_back(struct sm2_ep *ep,
 	sm2_fifo_write(ep, xfer_entry->hdr.sender_gid, xfer_entry);
 }
 
+static inline const char *sm2_print_ptr(struct sm2_ep *ep, void *ptr)
```

Comment:
Please add more of a description in the commit explaining what exactly this does (who owns the ptr, what look up is it doing, etc)

Reply:
That's fine but please put it in both places

Reply:
done